components/multiselect/multiselect.tsx

```diff
@@ -420,16 +462,16 @@ const styles = {
             backgroundColor: 'hsl(0,0%,100%)',
         };
     },
-    multiValue: (base) => {
+    multiValue: (base: React.CSSProperties) => {
         return {
             ...base,
-            whiteSpace: 'nowrap',
+            whiteSpace: 'nowrap' as 'nowrap',
```

Comment:
why is this `as` needed here?

Reply:
TS was complaining because the string `"nowrap"` does not match the type alias `'nowrap'`.  I'll see if I can find a workaround because I agree it's gross.